### ISSUE TITLE
Cant use anything under CPU tuning. 

### ISSUE THREAD

**Issue by nikhilpanchal1** [aredden/electron-G14Control#75]
Thanks for the amazing tool but I can't use anything under CPU tuning, all values fail to apply or save.
I cant apply plans either. They fail to apply and I doubt if my lappy is switching plans that I have set for AC and DC. 
Im on g14 2021, with 3060 and r9 5900hs

**Comment by aredden** [comment]
Yeah, this version of g14control requires the 2020 g14. Although for CPU related features, you can find the RyzenAdj executable in the g14control directory and replace it with a newer version of RyzenAdj which has support for zen 3 CPUs. Then you would be able to use it.

**Comment by aredden** [comment]
Also, the location of the RyzenAdj executable is here: `G14ControlV2/resources/extraResources/ryzenadj.exe`. Make sure it has the same name as the previous RyzenAdj executable, since G14Control grabs it using the name. Here is where the RyzenAdj exe can be downloaded.. https://github.com/FlyGoat/RyzenAdj/releases. I think v0.8.2, `ryzenadj-win64.zip` is what you are looking for. Put all the files in that folder into the extraResources folder and rename the .exe if it's different. Then you should be good to go :).

**Comment by hasan-hasanov** [comment]
@aredden Thank you, I was just about to open the same issue as @lenixxq. After your instructions everything worked perfectly.

**Comment by lenixxq** [comment]
> Also, the location of the RyzenAdj executable is here: `G14ControlV2/resources/extraResources/ryzenadj.exe`. Make sure it has the same name as the previous RyzenAdj executable, since G14Control grabs it using the name. Here is where the RyzenAdj exe can be downloaded.. https://github.com/FlyGoat/RyzenAdj/releases. I think v0.8.2, `ryzenadj-win64.zip` is what you are looking for. Put all the files in that folder into the extraResources folder and rename the .exe if it's different. Then you should be good to go :).

Thanks a lot man. My g14 2021 on armoury crate was god awful. Thanks A LOT for your work. I loved dGaming mode. it gave me 70watt tdp gaming back. after months. Any tips on what to do when you push update to the software?
